generators/client/templates/react/src/main/webapp/app/modules/administration/user-management/user-management.tsx

```diff
@@ -20,11 +20,13 @@ import * as React from 'react';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import { Button } from 'reactstrap';
+import { Pagination } from 'react-bootstrap';
```

Comment:
use https://reactstrap.github.io/components/pagination/ please

Reply:
ok I didn't see that for the pagination

Reply:
but it seems that with https://reactstrap.github.io/components/pagination/ we need to implement the logic of the pagination unlike react-bootstrap which provides a simple component

Reply:
@deepu105 since today there is a message on this page https://react-bootstrap.github.io/ which prevents that the project is under active development, and APIs will change so what do you think ?

Reply:
That message was always there and I have been following related issue for sometime, seems like there is no much progress and they still dont have BS4 in roadmap so for me `react-bootstrap` is a strict no for us and should stick to `reactstrap` implementing our pagination using reactstrap doesnt seem that difficult. Ofcourse the componenet structure is different but its not a big deal. If you want you can write a wrapper on top of reactstrap to make a simple component